ui/src/app/features/create-dialog/create-dialog.component.html

```diff
@@ -14,7 +14,7 @@
   ~ permissions and limitations under the License.
   -->
 
-<h1 mat-dialog-title>{{'common.create' | translate}} {{data.entityType}}</h1>
```

Comment:
why {{data.entityType}} was removed?

Reply:
> why {{data.entityType}} was removed?

entityType: 'model', this interpolation gives the word "model", and as it was mentioned in the task, we have to remove it from the dialog.